integration/all/top_tcp_test.go

```diff
@@ -93,27 +64,67 @@ func TestTopTcp(t *testing.T) {
 		RunTestSteps(commandsPostTest, t, WithCbBeforeCleanup(PrintLogsFn(ns)))
 	})
 
+	var extraArgs string
+	expectedEntry := &toptcpTypes.Stats{
+		Comm:      "curl",
+		IPVersion: 4,
+		SrcEndpoint: eventtypes.L4Endpoint{
+			L3Endpoint: eventtypes.L3Endpoint{
+				Addr:    "127.0.0.1",
+				Version: 4,
+			},
+		},
+		DstEndpoint: eventtypes.L4Endpoint{
+			L3Endpoint: eventtypes.L3Endpoint{
+				Addr:    "127.0.0.1",
+				Version: 4,
+			},
+			Port: 80,
+		},
+	}
+
+	switch DefaultTestComponent {
+	case IgTestComponent:
+		// TODO: Filter by namespace to avoid interferences with events from other
+		// tests. In the meanwhile, given that we are generating events by writing
+		// into a file:
```

Comment:
I think this comment is not relevant for this case. (In this one we aren't writing to files but creating tcp connections).